pkg/detectors/slackwebhook/slackwebhook.go

```diff
@@ -24,7 +24,10 @@ var _ detectors.Detector = (*Scanner)(nil)
 var (
 	defaultClient = common.SaneHttpClient()
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
-	keyPat = regexp.MustCompile(`(https://hooks\.slack\.com/services/T[A-Z0-9]+/B[A-Z0-9]+/[A-Za-z0-9]{23,25})`)
+	keyPats = map[string]*regexp.Regexp{
+		"Slack Service Web Hook":   regexp.MustCompile(`(https://hooks\.slack\.com/services/T[A-Z0-9]+/B[A-Z0-9]+/[A-Za-z0-9]{23,25})`),
+		"Slack Workflow Web Hook ": regexp.MustCompile(`(https://hooks\.slack\.com/workflows/T[A-Z0-9]+/A[A-Z0-9]+/[0-9]{17,19}/[A-Za-z0-9]{23,25})`),
```

Comment:
question: I'm not intimately familiar with slack webhooks, but when I go into our slack to create a workflow the "Web request URL" I am getting looks like `https://hooks.slack.com/triggers/TEL4..../69.../..`. Is this because Slack updated the format of the URL or i'm just setting it up in the wrong place.

Reply:
No you're right, this format seems to be a legacy one already.

Reply:
Great, thanks for confirming.